.github/workflows/pull-request.yml

```diff
@@ -34,6 +34,25 @@ jobs:
         run: |
           curl -k "$CLUSTER/apis/forklift.konveyor.io/v1beta1/namespaces/konveyor-forklift/providers" --header "Authorization: Bearer $TOKEN"
 
+      # Run e2e sanity
+      - name: Run e2e sanity suite
+        env:
+          OVIRT_USERNAME: admin@internal
+          OVIRT_PASSWORD: 123456
+          OVIRT_URL: https://fakeovirt.konveyor-forklift:30001/ovirt-engine/api
+          OVIRT_CACERT: /home/runner/work/_actions/kubev2v/forkliftci/main/ovirt/e2e_cacert.cer
+          STORAGE_CLASS: standard
+          OVIRT_VM_ID: 31573c08-717b-43e0-825f-69a36fb0e1a1
+        run: |
+          GOPATH=${GITHUB_WORKSPACE}/go make e2e-sanity
+
+      # Push code coverage using Codecov Action
+      - name: Push code coverage to Codecov
+        uses: codecov/codecov-action@v3
+        with:
+          fail_ci_if_error: false # see https://github.com/codecov/codecov-action/issues/598
```

Comment:
I don't see out previous thread on this part but I remember seeing that you say it's on a different job - still, so why do we need to post this twice? maybe I'm missing something but this part is under ci_setup although it's not about setup, should we replace `Run test suite` below with this?

Reply:
the `Run test suite` is under `build` job. you can see the steps we are doing there. we *don't* deploy kind of anything like that so i don't think it's suitable to the test. maybe i'm missing something but this is what i understand.

Reply:
I see, that makes sense - so how about having a better differentiation between unit tests and integration (e2e) tests here and in the makefile? it would make it a bit more clear..